src/helix-ui/styles/elements/hx-icon.less

```diff
@@ -12,4 +12,15 @@ hx-icon {
     fill: currentColor;
     stroke: none;
   }
+
+}
+
+.hxInline > hx-icon,
+hx-icon.hxInline {
```

Comment:
Consider going the other way around where not inline is the exception since most icons are inline.